peer.go

```diff
@@ -368,6 +371,10 @@ out:
 		case *lnwire.CommitSignature:
 			isChanUpate = true
 			targetChan = msg.ChannelPoint
+		// ROUTING ADDED
+		case *messages.NeighborAckMessage, *messages.NeighborHelloMessage, *messages.NeighborRstMessage, *messages.NeighborUpdMessage, *messages.RoutingTableRequestMessage, *messages.RoutingTableTransferMessage:
```

Comment:
Each of the target types in this case should appear on individual lines for display purposes.